data/mods/blazemod/recipes/blaze_other_recipes.json

```diff
@@ -102,7 +102,7 @@
     "components": [ [ [ "hard_plate", 10 ] ], [ [ "chain", 8 ] ] ]
```

Comment:
Small but important change here. With this, I've also removed all other mentions to `turretframe`, which means I'll need to add migration code. Beyond that, allows the vanilla turret_mount to be crafted, and brings all of the content in blazemod to parity only using vanilla turret mounts. Space anomaly left unchanged, just separated out to here.